base/src/main/java/io/spine/base/Time.java

```diff
@@ -140,21 +140,37 @@ default ZoneId currentZone() {
     @VisibleForTesting
     static class SystemTimeProvider implements Provider {
 
+        public static final int NANOSECONDS_IN_MILLISECOND = 1_000_000;
```

Comment:
Let's hide this constant.

Also, please look around. I would think there is a constant like this somewhere, already defined for you.